core_data_modules/traced_data.py

```diff
@@ -0,0 +1,154 @@
+import hashlib
+import json
+from deprecation import deprecated
+
+import six
+
+from core_data_modules.views.traced_data_items_view import _TracedDataItemsView
+from core_data_modules.views.traced_data_keys_view import _TracedDataKeysView
+from core_data_modules.views.traced_data_values_view import _TracedDataValuesView
+
+
+class SHAUtils(object):
```

Comment:
Please can you move this to the bottom of the file?

Reply:
Moved to util/sha_utils.py